rest-api-spec/src/main/resources/rest-api-spec/api/search.json

```diff
@@ -56,6 +56,10 @@
         "type":"string",
         "description":"The field to use as default where no field prefix is given in the query string"
       },
+      "fields": {
```

Comment:
I *think* this means that we take `fields` as a parameter on the url for search. But I don't think we added that. Which is fine by me.

Reply:
Oops, I'll remove this. I also think it's okay to not support `fields` as a URL parameter.